examples/seq2seq/finetune_trainer.sh

```diff
@@ -0,0 +1,12 @@
+# Add parent directory to python path to access lightning_base.py
+export PYTHONPATH="../":"${PYTHONPATH}"
```

Comment:
I think we won't need this for `finetune_trainer.py`

Reply:
feel free to delete!